### PR TITLE
RPRBLND-2079: Blender crash with AOV and Hybrid Pro

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Blender crash with AOV and Hybrid Pro.

### EFFECT OF CHANGE
RPR Passe panel shows only AOVs that are supported by the current Render Quality.

### TECHNICAL STEPS
Added 'hybrindpro' key to aovs_info, describes the support of pass by HybridPro engine.
Improved RPR_VIEWLAYER_PT_aovs, do_register_pass ,  export_aovs with the condition.
Small code improvements.

### CODE REVIEW

**Comment by bsavery** [comment]
IMO this should be handled at the RPR / Hybrid pro level, NOT worked around in the plugin code, as this leads to ugly solutions that never get fixed.  

Or at worst done in the RPR python binding for hybrid.

File: src/rprblender/properties/view_layer.py [hunk 1]
```diff
@@ -247,162 +248,194 @@ class RPR_ViewLayerProperites(RPR_Properties):
         {
             'rpr': pyrpr.AOV_COLOR,
             'name': "Combined",
-            'channel': 'RGBA'
+            'channel': 'RGBA',
+            'hybridpro': True
```
Comment (bnagirniak) [inline src/rprblender/properties/view_layer.py:252]: instead of adding 'hybridpro' option better all list variable with unsupported AOVs in pyhybridpro.py

**Review by bnagirniak — CHANGES_REQUESTED** [review]
Add warning with unsupported AOV in hybridpro

**Comment by VascoPi** [comment]
Oh I understand, I'll make another solution. Reported trace to HybridPro team, I wait for feedback.

**Comment by VascoPi** [comment]
Moved to https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/486

**Pull request closed without merging**